content/events/workshop-amsterdam-2020-03-30.md

```diff
@@ -0,0 +1,55 @@
+---
+title: KubeCon EU  Pre-day Workshop
+url: /events/workshop-amsterdam-2020-03-30
```

Comment:
Is this necessary? If we are just going to match the ~filename, shouldn't we just use whatever built-in support Hugo has for referring to the current page's filename?

Reply:
Yup, you're right, good catch.